### PR TITLE
delete games

### PR DESCRIPTION
**Pull request by reneesandefur**

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -9,7 +9,13 @@ def update
     @game.update_attributes(game_params)
     redirect_to_root_path
   end
-
+  
+  def destroy
+  end
+    @game = Game.find(params[:id])
```
Comment (cldambrosio) [inline app/controllers/games_controller.rb]: The logic should go between def / end ?

Comment (reneesandefur) [reply app/controllers/games_controller.rb]: haha yes...i don't know what happened there. i updated

Comment (cldambrosio) [reply app/controllers/games_controller.rb]: ;-D even though I was pretty sure of this it feels weird to review other's code!

File: app/controllers/games_controller.rb [hunk 2]
```diff
@@ -9,7 +9,14 @@ def update
     @game.update_attributes(game_params)
     redirect_to_root_path
   end
-
+  
+  def destroy
+    @game = Game.find(params[:id])
+      return not_found(:forbidden) if @game.user != current_user
+    @game.destroy
+    redirect_to_root_path
```
Comment (cldambrosio) [inline app/controllers/games_controller.rb]: I didn't catch this before: you need a space between `redirect_to` and `root_path`

**Pull request merged**